pylinkvalidator/crawler.py

```diff
@@ -828,6 +828,18 @@ def execute_from_config(config, logger):
     if not config.start_urls:
         raise Exception("At least one starting URL must be supplied.")
 
+    if config.options.allow_insecure_content:
```

Comment:
well, let's hope nobody uses pylinkvalidator as a library with this option :-) But that's fine, the main use is through the command line.